core/src/it/scala/io/km8/core/kafka/KafkaProducerSpec.scala

```diff
@@ -0,0 +1,57 @@
+package io.km8.core.kafka
+
+import zio.*
+import zio.blocking.Blocking
+import zio.clock.Clock
+import zio.test.*
+import zio.test.Assertion.*
+
+import com.dimafeng.testcontainers.KafkaContainer
+import io.km8.common.*
+import io.km8.core.kafka.KafkaProducer.KafkaProducer
+import org.apache.kafka.clients.consumer.OffsetResetStrategy
+
+object KafkaProducerSpec extends DefaultRunnableSpec:
+
+  private val consumerLayer: ZLayer[Has[KafkaContainer], Nothing, Has[KafkaConsumer]] =
+    Clock.live ++ Blocking.live ++ it_layers.clusterConfig(clusterId = "cluster_id") >>> KafkaConsumer.liveLayer
+
+  private val producerLayer: ZLayer[Has[KafkaContainer], Nothing, KafkaProducer] =
+    Blocking.live ++ it_layers.clusterConfig(clusterId = "cluster_id") >>> KafkaProducer.liveLayer
+
+  val specLayer: ZLayer[Has[KafkaContainer], Nothing, Has[KafkaConsumer] & KafkaProducer] =
+    consumerLayer ++ producerLayer
+
+  override def spec: ZSpec[_root_.zio.test.environment.TestEnvironment, Any] =
+    mainSpec
+      .provideSomeLayer[environment.TestEnvironment & Has[KafkaContainer]](specLayer)
+      .provideCustomLayerShared(Blocking.live >>> it_layers.kafkaContainer)
+
+  private val mainSpec =
+    suite("Kafka services")(
+      testM("KafkaProducer sends a message and KafkaConsumer reads it correctly ") {
+
+        val io: ZIO[Has[KafkaConsumer] with KafkaProducer, Throwable, String] = for {
```

Comment:
For some reason, the response never comes and the test doesn't finish. Under investigation...

Reply:
I think it's because of `filterKeyword = "*",`, leave it empty string